### PR TITLE
Icons: Carrot & Drumstick

### PR DESCRIPTION
**Pull request by Andreto**
image
image

### CODE REVIEW

**Comment by ericfennis** [comment]
Nice I love the style and simplicity.
Can you give small description for what kind of use cases we can use these icons?

**Comment by Andreto** [comment]
@ericfennis I created these for use on a website for my school's lunch menu. I wanted some icons to represent the ordinary meal and the vegetarian alternative.

**Comment by ericfennis** [comment]
@Andreto Thanks for sharing, I think this is great use case. I can imagine that you can use this as well in restaurant websites or "menu" applications as well.

**Review by ericfennis — APPROVED** [review]

File: icons/drumstick.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg
```
Comment (locness3) [inline icons/drumstick.svg:1]: Is this supposed to be called a "drumstick"? @Andreto

Comment (Andreto) [reply icons/drumstick.svg:1]: I think that's what it's called but of i have a better name it then we could change it.

**Pull request merged**